e2e/tests/stress_cpu.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Documentation    Negative Test Cases
+Resource    ../keywords/common.resource
+Resource    ../keywords/node.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/workload.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+
+Stress Volume Node CPU When Replica Is Rebuilding
+    Given Create a volume with 5 GB and 3 replicas
+    And Write data to the volume
+
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
+        When Delete replica on volume node to trigger replica rebuilding
+        And Stress the CPU of all volume nodes
+
+        Then Wait until replica on volume node rebuilt
+        And Check data is intact
+    END
+
+Stress Volume Node CPU When Volume Is Online Expanding
+    @{data_checksum_list} =    Create List
```

Comment:
Would it be possible if there's a more elegant way to deal with this instead of having lots of variables in resource layer? WDYT?

Reply:
I think it's fine for now. We can make improvements later when we have more cases to determine what should be included in the variable section.

Reply:
Variable section doesn't seems to be helpful. I'm afraid that setting variable (list) straightforwardly without a well-designed mechanism will bloat the code fast and make things hard to control when we deal with more complex test cases. Like the following test case:

```
create deployment
write data to deployment and create snapshot 0
write data to deployment and create snapshot 1
write data to deployment and create snapshot 2
check deployment data
revert to snapshot 1 and check deployment data
revert to snapshot 0 and check deployment data
create statefulset
write data to statefulset and create snapshot 0
write data to statefulset and create snapshot 1
write data to statefulset and create snapshot 2
check statefulset data
revert to snapshot 1 and check statefulset data
revert to snapshot 0 and check statefulset data
```

How can we get the correct checksum from a single checksum list contains checksums from different workloads and different snapshots?

Reply:
You're probably right. Using a variable section might not be the best approach. We can improve this later when we're dealing with more complex cases.

Reply:
We probably should figure out this now since we're defining the framework, otherwise when team members or contributors participate in, they'll follow the awkward pattern to write more test cases and make our lives difficult.

Reply:
Let's handle in https://github.com/longhorn/longhorn/issues/7034.

Reply:
See if we can handle in Let's address this in https://github.com/longhorn/longhorn/issues/7034.